docs/_guides/codewind-eclipse-quick-guide.md

```diff
@@ -37,61 +37,61 @@ The Codewind installation pulls the following images that form the Codewind back
 
 ### Creating an Appsody project
 Appsody helps you develop containerized applications and removes the burden of managing the full software development stack. If you want more context about Appsody, see the [Appsody welcome page](https://appsody.dev/docs).
 
-1. Right-click `Local [Running]` under `Codewind` in the `Codewind` tab.
-2. Select `+ Create New Project...`
+1. Right-click **Local [Running]** under **Codewind** in the **Codewind** tab.
+2. Select **+ Create New Project...**
     * **Note:** Make sure that Docker is running. Otherwise, you get an error.
-3. Name your project `appsody-calculator`.
-4. Under `Template`, select `Appsody Open Liberty default template`. 
-    * If you don't see an Appsody template, select the `Manage Template Sources...` link at the end of the window.
-    * Select the `Appsody Stacks - incubator` checkbox.
-    * Click `OK`.
+3. Name your project **appsody-calculator**.
+4. Under **Template**, select **Appsody Open Liberty default template**. 
+    * If you don't see an Appsody template, select the **Manage Template Sources...** link at the end of the window.
+    * Select the **Appsody Stacks - incubator** checkbox.
+    * Click **OK**.
     * The templates are refreshed, and the Appsody templates are available.
-5. Click `Finish`.
-    * To monitor your project's progress, right-click on your project, and select `Show Log Files`.
-    * Select `Show All`. Then, a `Console` tab is displayed where you see your project's build logs.
+5. Click **Finish**.
+    * To monitor your project's progress, right-click on your project, and select **Show Log Files**.
+    * Select **Show All**. Then, a **Console** tab is displayed where you see your project's build logs.
 
-Your project is displayed in the `Local [Running]` section. The progress for creating your project is tracked next to the project's name.
+Your project is displayed in the **Local [Running]** section. The progress for creating your project is tracked next to the project's name.
 
 Your project is complete when you see that your project is running and its build is successful.
 
 ### Accessing the application endpoint in a browser
 
 1. Return to your project under the Codewind tab.
-2. Right-click your project and select `Open Application`.
+2. Right-click your project and select **Open Application**.
 
 Your application is now opened in a browser, showing the welcome to your Appsody microservice page.
 
 ### Adding a REST service to your application
 
 1. Go to your project's workspace under the Project Explorer tab.
-2. Go to `Java Resources->src/main/java->dev.appsody.starter`.
-3. Right-click `dev.appsody.starter` and select `New->Class`.
-4. Create a Class file, name it `Calculator.java`, and select `Finish`. This file is your JAX-RS resource.
+2. Go to `Java Resources>src/main/java>dev.appsody.starter`.
+3. Right-click **dev.appsody.starter** and select **New>Class**.
```

Comment:
If "New" and "Class" are separate selections: `**New**>**Class**
If the whole name is "New>Class", then you can leave it as-is.